src/mscorlib/model.xml

```diff
@@ -9526,11 +9529,98 @@
+      <Member Name="AddValue(System.String,System.UInt16)" />
+      <Member Name="AddValue(System.String,System.Int32)" />
+      <Member Name="AddValue(System.String,System.UInt32)" />
+      <Member Name="AddValue(System.String,System.Int64)" />
+      <Member Name="AddValue(System.String,System.UInt64)" />
+      <Member Name="AddValue(System.String,System.Single)" />
+      <Member Name="AddValue(System.String,System.Double)" />
+      <Member Name="AddValue(System.String,System.Decimal)" />
+      <Member Name="AddValue(System.String,System.DateTime)" />
+      <Member Name="GetValue(System.String,System.Type)" />
+      <Member Name="GetBoolean(System.String)" />
+      <Member Name="GetChar(System.String)" />
+      <Member Name="GetSByte(System.String)" />
+      <Member Name="GetByte(System.String)" />
+      <Member Name="GetInt16(System.String)" />
+      <Member Name="GetUInt16(System.String)" />
+      <Member Name="GetInt32(System.String)" />
+      <Member Name="GetUInt32(System.String)" />
+      <Member Name="GetInt64(System.String)" />
+      <Member Name="GetUInt64(System.String)" />
+      <Member Name="GetSingle(System.String)" />
+      <Member Name="GetDouble(System.String)" />
+      <Member Name="GetDecimal(System.String)" />
+      <Member Name="GetDateTime(System.String)" />
+      <Member Name="GetString(System.String)" />
+      <Member MemberType="Property" Name="FullTypeName" />
+      <Member MemberType="Property" Name="AssemblyName" />
+      <Member MemberType="Property" Name="MemberCount" />
+      <Member MemberType="Property" Name="ObjectType" />
+      <Member MemberType="Property" Name="IsFullTypeNameSetExplicit" />
+      <Member MemberType="Property" Name="IsAssemblyNameSetExplicit" />
+    </Type>
+    <Type Name="System.Runtime.Serialization.SerializationInfoEnumerator">
+      <Member Name="MoveNext" />
+      <Member Name="Reset" />
+      <Member Name="System.Collections.IEnumerator.get_Current" />
+      <Member MemberType="Property" Name="Current" />
+      <Member MemberType="Property" Name="Name" />
+      <Member MemberType="Property" Name="Value" />
+      <Member MemberType="Property" Name="ObjectType" />
+    </Type>
+    <Type Name="System.Runtime.Serialization.ISerializable">
+      <Member Name="GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
+    </Type>
+    <Type Name="System.Runtime.Serialization.IFormatterConverter">
+      <Member Name="Convert(System.Object,System.Type)" />
+      <Member Name="Convert(System.Object,System.TypeCode)" />
+      <Member Name="ToBoolean(System.Object)" />
+      <Member Name="ToChar(System.Object)" />
+      <Member Name="ToSByte(System.Object)" />
+      <Member Name="ToByte(System.Object)" />
+      <Member Name="ToInt16(System.Object)" />
+      <Member Name="ToUInt16(System.Object)" />
+      <Member Name="ToInt32(System.Object)" />
+      <Member Name="ToUInt32(System.Object)" />
```

Comment:
>ToUInt32 [](start = 20, length = 8)

nit, it's nice to use alpha order because it makes it easier to diff with the tooling generated one. fine either way though.

    <Type Name="System.Runtime.Serialization.IFormatterConverter">
      <Member Name="Convert(System.Object,System.Type)" />
      <Member Name="Convert(System.Object,System.TypeCode)" />
      <Member Name="ToBoolean(System.Object)" />
      <Member Name="ToByte(System.Object)" />
      <Member Name="ToChar(System.Object)" />
      <Member Name="ToDateTime(System.Object)" />
      <Member Name="ToDecimal(System.Object)" />
      <Member Name="ToDouble(System.Object)" />
      <Member Name="ToInt16(System.Object)" />
      <Member Name="ToInt32(System.Object)" />
      <Member Name="ToInt64(System.Object)" />
      <Member Name="ToSByte(System.Object)" />
      <Member Name="ToSingle(System.Object)" />
      <Member Name="ToString(System.Object)" />
      <Member Name="ToUInt16(System.Object)" />
      <Member Name="ToUInt32(System.Object)" />
      <Member Name="ToUInt64(System.Object)" />
    </Type>